graphql-dgs-spring-graphql/src/main/kotlin/com/netflix/graphql/dgs/springgraphql/autoconfig/DgsSpringGraphQLEnvironmentPostProcessor.kt

```diff
@@ -28,7 +29,19 @@ class DgsSpringGraphQLEnvironmentPostProcessor : EnvironmentPostProcessor {
     ) {
         val properties = mutableMapOf<String, Any>()
 
-        properties["spring.graphql.schema.introspection.enabled"] = environment.getProperty("dgs.graphql.introspection.enabled") ?: true
+        if (environment.getProperty("spring.graphql.schema.introspection.enabled") != null &&
```

Comment:
consider using local vars or better constants for property names (idk what's the probability of ever having to rename `spring.graphql.schema.introspection.enabled` but updating 4 strings just in this file is error-prone)

Reply:
done